cogs/ahk/ahk.py

```diff
@@ -263,6 +257,22 @@ async def ahk(self, ctx, *, code):
 
 		await ctx.send(out)
 
+		return stdout, time
+
+	@commands.command()
+	@commands.cooldown(rate=1.0, per=5.0, type=commands.BucketType.user)
+	async def rlx(self, ctx, *, code):
+		'''Compile and run Relax code through CloudAHK. Example: `rlx define i32 Main() {return 20}`'''
```

Comment:
maybe geek can auto-insert the main function boilerplate on his side?

Reply:
He probably could, but it would need to be "smart" enough that code which defines multiple functions wouldn't end up all wrapped in the `Main` function. In my opinion, it would be lots of work for very little gain.